.github/workflows/documentation.yaml

```diff
@@ -18,32 +18,26 @@ jobs:
       matrix:
         python-version: ["3.10"]
 
-    
     steps:
       - name: Checkout code
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
 
       - name: Set up Python
-        uses: actions/setup-python@v3
+        uses: actions/setup-python@v5
```

Comment:
> [!NOTE]
> These were a bit old, so I updated them. It would be nice to use `renovatebot` or `dependabot`; probably after this PR; we can work on that